pkg/gds/gds.go

```diff
@@ -294,7 +294,7 @@ func (s *GDS) Lookup(ctx context.Context, in *api.LookupRequest) (out *api.Looku
 	case in.Id != "":
 		// TODO: add registered directory to lookup
 		if vasp, err = s.db.RetrieveVASP(in.Id); err != nil {
-			log.Warn().Err(err).Str("id", in.Id).Str("registered_directory", in.RegisteredDirectory).Msg("could not find VASP by ID")
+			log.Debug().Err(err).Str("id", in.Id).Str("registered_directory", in.RegisteredDirectory).Msg("could not find VASP by ID")
```

Comment:
We were definitely feeling alert fatigue with these so thanks for taking the opportunity to make this change!